### ISSUE TITLE
longer list

### ISSUE THREAD

**Issue by wpoely86** [timstaley/jabref-astro-abbreviations#1]
Hi,

Inspired by your list, I've pulled all the abbreviations from web of science into a Jabref file. Feel free to use: https://github.com/wpoely86/jabref-abbreviations

Thanks for the idea!

**Comment by timstaley** [comment]
Hey, thanks for the link! Unfortunately, the preferred abbreviations tend to be journal specific, so (unless they converge on a single standard) per-journal lists are required. But it's useful to have the Web of Science list on stand-by if it's required in future.

**Comment by wpoely86** [comment]
Don't they all use the [ISO 4](https://en.wikipedia.org/wiki/ISO_4) standard? 

I've merged the list upstream: JabRef/jabref#51 so in the future, just use the new version of Jabref :wink:

**Comment by timstaley** [comment]
Unfortunately not, see e.g. http://www.oxfordjournals.org/our_journals/mnras/for_authors/

```
The following simplified abbreviations are used for frequently used journals, 
as in the examples above. For journals not on this list, use the IAU standard 
abbreviations published on the IAU website.

A&A: Astronomy and Astrophysics
A&ARv: Astronomy and Astrophysics Review (the)
A&AS: Astronomy and Astrophysics Supplement Series
...
```

However, on closer inspection, it appears [Astronomy and Computing](http://www.elsevier.com/journals/astronomy-and-computing/2213-1337/guide-for-authors#47000) **does**, so this is useful in that regard at least:  "Journal names should be abbreviated according to the List of Title Word Abbreviations: http://www.issn.org/services/online-services/access-to-the-ltwa/."
